docs/key-issues/multiple_architectures.md

```diff
@@ -0,0 +1,259 @@
+# Platforms with multiple CPU architectures
+
+In addition to any ABI requirements, a binary is compiled for a CPU
+architecture. That CPU architecture defines the CPU instructions that can be
+issued by the binary.
```

Comment:
I think this is less clear than it could be. From my POV, anything baked into a compiled artefact is part of the ABI, including assumptions of what CPU architecture instruction set extensions are made use of. I'd say:

```suggestion
One important subset of ABI concerns is which CPU architecture a produced
binary artefact is expecting, resp. which CPU instructions from that
architecture the artifact has been compiled to use[^1]. Trying to use a binary
on a CPU not matching the target architecture specified during compilation
(or similarly, on a processor that does not support all of the instructions
the binary artifact expects to be available) will generally lead to crashes.

[^1] compare how the x86-64 architecture has various well-known extensions like SSE, SSE2, SSE3, AVX, AVX2, AVX512, etc.
```